src/applications/static-pages/static-pages-entry.js

```diff
@@ -119,7 +119,7 @@ if (burialPages.has(location.pathname)) {
   });
 }
 
-if (disabilityPages.has(location.pathname) && __BUILDTYPE__ !== 'production') {
+if (disabilityPages.has(location.pathname) && brandConsolidation.isEnabled()) {
   createDisabilityIncreaseApplicationStatus(store);
 }
```

Comment:
This should be enabled on all VA.gov domains, but not on Vets.gov? We can possibly just remove the Vets.gov page paths from the list of Disability pages. Either way, looks good 👍

Reply:
Right, exactly. I made sure with @djmunoz1 before making the change. Vets.gov will have the `Apply on eBenefits` button, but VA.gov will have the find-your-form wizard (which is what this component is).